wsd/DocumentBroker.cpp

```diff
@@ -4567,6 +4570,30 @@ void DocumentBroker::handleMediaRequest(std::string range,
     }
 }
 
+bool DocumentBroker::requestTileRendering(TileDesc& tile, bool forceKeyframe,
+                                          const std::chrono::steady_clock::time_point &now,
+                                          std::vector<TileDesc>& tilesNeedsRendering,
+                                          const std::shared_ptr<ClientSession>& session)
+{
+    bool allSamePartAndSize = true;
+    if (!tileCache().hasTileBeingRendered(tile, &now) || // There is no in progress rendering of the given tile
+        tileCache().getTileBeingRenderedVersion(tile) < tile.getVersion()) // We need a newer version
```

Comment:
I'm not so sure that version is the right thing here - but it can't do that much harm I think. Looks like we will re-render but send an empty delta.

Reply:
This is directly moved from `DocumentBroker::sendRequestedTiles`.

I assume it was thought to bump version (line below) so `getTileBeingRenderedVersion` will be equal with the next potential request while this one is pending.

That will ignore it and not allow to render the same twice. Then we still subscribe to it - so after finished render we will send it.

Reply:
Yep all good =)

Reply:
I checked the version thing and it was duplicated as I missed previous bump.
Followup: https://github.com/CollaboraOnline/online/pull/11171